README.md

```diff
@@ -44,18 +44,22 @@ await myToken.start() // load contract and connection into the class representin
 await myToken.transferTokenAmount('0xYourOtherAddress', 1); // transfer 1 token from your address to other address
 
 ```
-
 Please refer to the [`test/`](./test/models) folder to read further usage examples of the various contracts available.
 
-## Generating documentation
+## Documentation 
+
+[Guides](https://docs.bepro.network/sdk-documentation/start-building/how-to-guides)
+[SDK Documentation](https://bepronetwork.github.io/bepro-js/)
```

Comment:
This SDK documentation is wrong, as it will point to the V1 of the SDK; It should point to a distribution of the `npm run docs`

Reply:
Created a story to deploy the docs.